*** tests/A-01-10-M-ok.out	2020-12-01 16:55:11.000000000 +0000
--- tests/A-01-10-M-ok.outhyp	2020-12-11 19:05:38.559177933 +0000
***************
*** 10,16 ****
  9 - Menu de Consultas
  10 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Menu Principal
  1 - Abrir
  2 - Guardar
  3 - Mostrar Data
--- 10,17 ----
  9 - Menu de Consultas
  10 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Abrir: Operação inválida: Problema ao abrir 'app04.woo'.
! Menu Principal
  1 - Abrir
  2 - Guardar
  3 - Mostrar Data
***************
*** 29,35 ****
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: C1|R2-D2|somewhere|NORMAL|0|0
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
--- 30,36 ----
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: Mostrar cliente: Operação inválida: O cliente 'C1' não existe.
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
***************
*** 37,44 ****
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: C2|C3PO|with R2-D2|NORMAL|0|0
! BARGAIN|H2|2
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
--- 38,44 ----
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: Mostrar cliente: Operação inválida: O cliente 'C2' não existe.
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
