*** tests/A-17-07-M-ok.out	2020-12-01 16:55:11.000000000 +0000
--- tests/A-17-07-M-ok.outhyp	2020-12-11 19:05:51.779511117 +0000
***************
*** 40,49 ****
  3 - Permitir/inibir transacções
  4 - Consultar transacções
  0 - Sair
! Escolha uma opção: Identificador do fornecedor: 0|P2|2616|0
! 2H2|303
! 3O3|125
! 6M2|101
  Gestão de Fornecedores
  1 - Mostrar fornecedores
  2 - Registar
--- 40,46 ----
  3 - Permitir/inibir transacções
  4 - Consultar transacções
  0 - Sair
! Escolha uma opção: Identificador do fornecedor: woo.core.Order@6f496d9f
  Gestão de Fornecedores
  1 - Mostrar fornecedores
  2 - Registar
