*** tests/A-18-17-M-ok.out	2020-12-01 16:55:11.000000000 +0000
--- tests/A-18-17-M-ok.outhyp	2020-12-11 19:05:56.115619606 +0000
***************
*** 10,16 ****
  9 - Menu de Consultas
  10 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Menu Principal
  1 - Abrir
  2 - Guardar
  3 - Mostrar Data
--- 10,17 ----
  9 - Menu de Consultas
  10 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Abrir: Operação inválida: Problema ao abrir 'selection.woo'.
! Menu Principal
  1 - Abrir
  2 - Guardar
  3 - Mostrar Data
***************
*** 28,34 ****
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador do cliente: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
  3 - Registar encomenda
--- 29,36 ----
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador do cliente: Data limite de pagamento: Identificador do produto: Quantidade: Registar venda: Operação inválida: O produto 'B' não existe.
! Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
  3 - Registar encomenda
***************
*** 64,76 ****
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador da transacção: Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador da transacção: 2|R2|B|5|2000|2120|4|7
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
--- 66,79 ----
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador da transacção: Pagar: Operação inválida: A transacção '2' não existe.
! Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador da transacção: Visualizar: Operação inválida: A transacção '2' não existe.
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
***************
*** 96,102 ****
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|2600|2660
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
--- 99,105 ----
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: Mostrar cliente: Operação inválida: O cliente 'R2' não existe.
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
